Sources/Conduit/Auth/Cryptography/Symmetric/KeychainHybridKeyProvider.swift

```diff
@@ -57,10 +57,7 @@ public final class KeychainHybridKeyProvider: HybridKeyProvider {
             guard let errorCode = error.flatMap({ LAError.Code(rawValue: $0.code) }) else {
                 return false
             }
-            if #available(OSX 10.13, iOS 11, *) {
```

Comment:
Since we declare compatibility with iOS 9 in Package.swift, I don't think we can remove this availability check.